src/java-runtime/assets/index.ts

```diff
@@ -58,6 +58,7 @@ declare function acquireVsCodeApi(): any;
 const vscode = acquireVsCodeApi();
 
 function requestJdkInfo(jdkVersion: string, jvmImpl: string) {
+  console.log(`request ${jdkVersion} ${jvmImpl}`);
```

Comment:
is the console.log on purpose?

Reply:
omg, seems like the merge had some problem